components/src/core/utility/utility.scss

```diff
@@ -0,0 +1,3 @@
+@mixin invert {
+    transform: scaleX(-1);
+}
```

Comment:
Quick question about this `@invert` mixin. Does this mean we're auto-rotating all icons that a user passes in? What is their recourse if they are using one that shouldn't be flipped? 

On the react-side, we are only auto-rotating the ones that we use internally and leaving it up to the end user to rotate the ones they want to flip.

Reply:
Since we're doing these rotations through css, if someone doesn't want their icon flipped they could override the transform that is happening, something like this:

```
<pxb-hero label="Battery" value="Full">
      <battery-progress
            style="transform: none"
            percent="90"
             size="36"
            pxb-primary></battery-progress>
</pxb-hero>
```